### PR TITLE
Implemented t, F, chisq distributions

### PR DESCRIPTION
**Pull request by cenonn**
I've added in the 3 requested distributions. I'm not sure how the other distributions were tested so I have not done any testing on the functionality yet.

### CODE REVIEW

File: symbulate/distributions.py [hunk 1]
```diff
@@ -363,6 +363,62 @@ def __init__(self, a, b, scale=None):
     def draw(self):
         return np.random.beta(self.a, self.b)
 
+class t(Distribution):
```
Comment (dlsun) [inline symbulate/distributions.py]: Avoid calling this t. (It's very possible the user will have a variable called t already.) Let's call it StudentT.

**Comment by dlsun** [comment]
Did you test out your code? Just re-install the package, open up a Jupyter notebook, and make sure that the three distributions work as you think they should.

**Comment by cenonn** [comment]
The setup file installs the package based on whatever is on the master branch. Since these changes are not on the master branch yet, how would I install a version with the changes?

**Comment by dlsun** [comment]
I think the setup file installs whatever files are currently in the symbulate/ directory. If you've done git checkout your-branch and are currently in that branch, then it should install the files in that branch, does it not?

**Comment by cenonn** [comment]
You are correct. I needed to update the \_\_init\_\_.py so it would import in the 3 new distributions. I will update the request with the changes after I test.

**Pull request merged**